index.html

```diff
@@ -0,0 +1,2059 @@
+                  <a class="h5-a" href="#">Best Maternity Hospitals 2020</a>
+                </h5>
+                <p class="card-text">
+                  The facilities cited by Newsweek are an elite group
+                  demonstrating excellence in maternity care. Check the full
+                  list here
+                </p>
+              </div>
+            </article>
+            <article class="top-story card wrap mt-5">
+              <div class="wrap">
+                <span
+                  class="wrap font-weight-bold badge-secondary text-primary px-3"
+                  >RANKINGS</span
+                >
+                <img
+                  src="https://d.newsweek.com/en/full/1603109/americas-best-realtors-2020.jpg?w=500&h=333&f=d564ac26330b80cd33b2b013a3e6fc8b"
+                  class="card-img-top"
+                  alt="walmart-store-front"
+                />
+              </div>
+              <div class="card-body">
+                <h5 class="card-title font-weight-bold">
+                  <a class="h5-a" href="#">America’s Best Realtors 2020</a>
+                </h5>
+                <p class="card-text">
+                  Newsweek partnered with REAL Trends to sort through the data
+                  and provide you with a list of America's Best Real Estate
+                  Professionals
+                </p>
+              </div>
+            </article>
+            <article class="top-story card wrap my-5">
+              <div class="wrap">
+                <img
+                  src="https://d.newsweek.com/en/full/1568509/newsweek-amplify.png?w=500&h=333&f=98c0ebfc554597997a0de3bd6159812a"
+                  class="card-img-top"
+                  alt="walmart-store-front"
+                />
+              </div>
+              <div class="card-body">
+                <h5 class="card-title font-weight-bold">
+                  <a class="h5-a" href="#"
+                    >Newsweek AMPLIFY Launches Strategic Partnership Solutions
+                    for Brands</a
+                  >
+                </h5>
+                <p class="card-text">
+                  Newsweek AMPLIFY, the company's performance marketing business
+                  unit, has launched a new program to empower digital
+                </p>
+              </div>
+            </article>
+          </div>
+          <!--End of survey & rankings section-->
+          <!--More Stories section-->
+          <section class="d-md-none gap">
+            <div class="more-stories mt-5">
+              <div class="h2-div font-weight-bold">MORE STORIES</div>
+              <hr />
+              <div class="media-1">
+                <img
+                  class="float-left mr-3"
+                  alt="64x64"
+                  src="https://d.newsweek.com/en/full/1626297/joe-biden-jill-dnc-2020.jpg?w=95&h=95&f=44299f0d2be506ecefd4079ee9206bdf"
+                  style="width: 95px; height: 95px;"
```

Comment:
- [ ] Please remove the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)